pkg/openfeaturetest/testprovider.go

```diff
@@ -0,0 +1,104 @@
+package openfeaturetest
```

Comment:
I guess we need to come up with a better place for this ? 🤔

Reply:
the question is, if we want this to be part of the sdk or rather put this as a separate package in the sdk-contrib - like we did in java for the junit extension.

Reply:
then to me this sounds like contrib would be the better place. @toddbaert ?

Reply:
Since there's no external deps, I think we should include this in the SDK.